docs/topics/serialization.txt

```diff
@@ -636,6 +636,29 @@ command line flags to generate natural keys.
     natural keys during serialization, but *not* be able to load those
     key values, just don't define the ``get_by_natural_key()`` method.
 
+.. versionchanged:: 6.1
```

Comment:
When 6.1 goes out of support, we remove any sections with this directive. But we don't want to lose your whole note here! Once your note is a small as possible, integrate it into the main text above. Then this `.. versionchanged::` bit can just be a single sentence e.g. "Support for opting-out of natural key serialization with an empty tuple was added."